Add more information on DB permissions (#398)
CleverCloud/documentation: content/doc/addons/mongodb/_index.md

**The managed databases provide an advantage if you want to avoid the complexity of installing and deploying databases** and if you want simple-to-use management and monitoring tools.

> ## 🔑 Rights and permissions

Reviewer: As this part is almost the same for different databases, maybe we should: 
- Keep a shortcode integration
- Use general language across databases 
- Explain what's specific to an add-on in a dedicated part

Author: I agree with @aurrelhebert since even in the similar text parts there are subtle differences between each database (like the specified db role / permissions)

> Add-ons are managed services, meaning that users have **standard access** to the database (**owner** of the database). Some operations like database and user creation, as well as some settings modifications aren't available by default. This ensures optimal performances and security for managed services as configured by Clever Cloud.

Reviewer: I think we should try to avoid repetitions of words such as `database` as much as we can (not simple on such parts of the documentation, but it makes it better to read)

Author: Ok I removed the repetition

> If you think your system might require more advanced administrative access, contact Clever Cloud support to explain your use case and we will work with you to find a solution.

Reviewer: ```suggestion
If your system require more advanced administrative access, [contact Clever Cloud support](https://console.clever-cloud.com/ticket-center-choice). We will try to find a solution that fit your needs.
```

Here is the list of actions that you won't be able to perform:
- Database administration (for example you won't be able to create new databases).
- Users administration (you won't be able to create other users than the one handled with our control plane, ie the base owner and read-only users).
- Server configuration update.
- Cluster creation
- Back-up frequency or retention control.

Please refer to Clever Cloud support if you want to perform one of these actions.

Authorized actions:

> - Manage collections (create, delete...)

Reviewer: This part could be specific to each add-on. Thus, here we can use: 

```suggestion
Refer to Clever Cloud support if you want to manage collections (create, delete, etc.).
```

Maybe `refer to` could be replaced by `Ask`

Author: done for all four files
